File: src/nomanssave/cC.java
package nomanssave;

import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;
import javax.swing.JOptionPane;

class cC implements ActionListener {
  cC(cy paramcy) {}
  
  public void actionPerformed(ActionEvent paramActionEvent) {
    String str = cy.c(this.gg).getText().trim();
    try {
      cy.d(this.gg).setText(str);
      ((cI)cy.e(this.gg).getModel()).a(cy.d(this.gg));
      cy.e(this.gg).setSelectionRow(0);
      cy.e(this.gg).setVisible(true);
      cy.f(this.gg).setVisible(false);
    } catch (eX eX) {
      JOptionPane.showOptionDialog(this.gg, "Error on line #" + eX.getLineNumber() + ": " + eX.getMessage(), "Error", 0, 0, null, new Object[] { "Cancel" }, null);
      cy.c(this.gg).setCaretPosition(eX.bD());
      cy.c(this.gg).requestFocus();
    } 
  }
}


/* Location:              E:\NMSSE_Source\NMSSaveEditor.jar!\nomanssave\cC.class
 * Java compiler version: 8 (52.0)
 * JD-Core Version:       1.1.3
 */

File: NMSSaveEditor/nomanssave/mixed/cC.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Windows.Forms;
using System.Globalization;

namespace NMSSaveEditor
{

public class cC : ActionListener {
   // $FF: synthetic field
   cy gg;

   cC(cy var1) {
      this.gg = var1;
   }

   public void actionPerformed(ActionEvent var1) {
      string var2 = cy.c(this.gg).GetText().Trim();

      try {
         cy.d(this.gg).SetText(var2);
         ((cI)cy.e(this.gg).GetModel()).a(cy.d(this.gg));
         cy.e(this.gg).setSelectionRow(0);
         cy.e(this.gg).SetVisible(true);
         cy.f(this.gg).SetVisible(false);
      } catch (eX var4) {
         JavaCompat.ShowOptionDialog(this.gg, "Error on line #" + var4.getLineNumber() + ": " + var4.getMessage(), "Error", 0, 0, (Icon)null, new Object[]{"Cancel"}, (Object)null);
         cy.c(this.gg).setCaretPosition(var4.bD());
         cy.c(this.gg).Focus();
      }

   }
}

}
